file-avail-s3.md

```diff
@@ -0,0 +1,235 @@
+    programs (aka "executables").
+2.  CHTC's [Intro to Running HTCondor Jobs](/helloworld.shtml)
+3.  CHTC's guide for [Typical File Transfer](/file-availability.shtml)
+
+Contents
+--------
+
+1.  [Policies and Intended Use](#1-policies-and-intended-use)
+2.  [Staging Large Data](#2-staging-large-data)
+3.  [Using Staged Files in a Job](#3-using-staged-files-in-a-job)
+	- [Accessing Large Input Files](#a-accessing-large-input-files)
+	- [Moving Large Output Files](#b-moving-large-output-files)
+4.  [Checking your Data Use and File Counts](#4-checking-your-data-use-and-object-counts)
+
+# 1. Policies and Intended Use
+
+> **USERS VIOLATING ANY OF THE POLICIES IN THIS GUIDE WILL
+> HAVE THEIR DATA STAGING ACCESS AND/OR CHTC ACCOUNT REVOKED UNTIL CORRECTIVE
+> MEASURES ARE TAKEN. CHTC STAFF RESERVE THE RIGHT TO REMOVE ANY
+> PROBLEMATIC USER DATA AT ANY TIME IN ORDER TO PRESERVE PERFORMANCE.**
+
+
+## A. Intended Use
+
+Our S3 data storage is only for input and output files that
+are individually too large to be managed by our other data movement
+methods, HTCondor file transfer or SQUID, and when these files are
+expected to be accessed outside of CHTC. This includes individual input files
+greater than 100MB and individual output files greater than 3-4GB.
+
+Users are expected to abide by this intended use expectation and follow the
+instructions for using S3 written in this guide (e.g. files placed
+in S3 should ALWAYS be listed in the submit file).
+
+## B. Access to S3 Data Storage
+
+Any one with a CHTC account whose data meets the intended use above
+can request access to CHTC's S3 data storage. A Research
+Computing Facilitator will review the request and follow up. If
+appropriate, S3 bucket creation will be enabled and a quota set on
+your account. Quotas are based on individual user needs; if a larger quota
+is needed, email chtc@cs.wisc.edu with your request.
+
+## C. User Data Management Responsibilities
+
+As with all CHTC file spaces:
+
+- **Keep copies**: Our S3 buckets are not backed up and have the
+possibility of data loss; keep copies of ANY and ALL data in S3
+buckets in another, non-CHTC location.
+- **Remove data**: We expect that users remove data from S3 buckets AS
+SOON AS IT IS NO LONGER NEEDED FOR ACTIVELY-RUNNING JOBS.
+- **Monitor usage and quota**: Your account has both a size and
+number of files quota that applies across all buckets owned by your
+account. Quota changes can be requested by emailing chtc@cs.wisc.edu.
+
+CHTC staff reserve the right to remove S3 buckets or revoke bucket
+creation permission at any time.
+
+## D. Data Access Within Jobs
+
+Data in a CHTC S3 bucket is available anywhere (including OSG) as long
+as the version of HTCondor running on the execute machines support
+jobs that use S3 URLs.
+
+Data in CHTC S3 buckets are owned by the user (or a set of users), and
+only the user's (or users') own jobs can access these files.
+
+# 2. Staging Large Data
```

Comment:
--> Staging Large Data in S3 Buckets?

Not sure if general or specific is better. :P

Reply:
Going with specific in https://github.com/CHTC/chtc-website-source/pull/56/commits/c1f2030c090714346d3cd7e43ce1cb7d44bcad1f